test/regress_dns.c

```diff
@@ -2437,6 +2441,8 @@ struct testcase_t dns_testcases[] = {
 
 	{ "client_fail_requests", dns_client_fail_requests_test,
 	  TT_FORK|TT_NEED_BASE|TT_NO_LOGS, &basic_setup, NULL },
+	{ "client_fail_waiting_requests", dns_client_fail_requests_test,
+	  TT_FORK|TT_NEED_BASE|TT_NO_LOGS, &basic_setup, (char*)"limit-inflight" },
```

Comment:
This new test doesn't pass on the current master branch and crashes if library is built with sanitizers.